docs/rules/require-optimization.md

```diff
@@ -21,23 +21,23 @@ Examples of **correct** code for this rule:
 
 ```js
 class YourComponent extends React.Component {
-	shouldComponentUpdate () {
-		return false;
-	}
+ shouldComponentUpdate () {
+  return false;
+ }
```

Comment:
this is one-space indentation, which even a spaces proponent would agree is worse than tabs :-) why can't this be tabs?

Reply:
This was autofixed by this rule: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md010

It looks like pretty much all of our other code samples use spaces instead of tabs, so this improve consistency.

I reconfigured the rule to use 2 spaces instead.

Reply:
i guess that's fine for now, thanks